_guide/_pages/interviews/technical.md

```diff
@@ -61,89 +61,29 @@ to produce and defend a desired solution. Preface by saying there’s no right a
-
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.1s7pt8lyer05">Tell me about how you have worked with a team to remediate a security issue.</a></p>
-
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.v2fk06y6uimr">Tell me about an innovative idea (i.e. a tool, practice, or policy) that you helped introduce.</a></p>
-
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.t6dedwrnx2k8">Describe a situation where you disagreed with someone about a security issue.</a></p>
-
-    </div>
-  <h4 class="usa-accordion__heading">
-    <button class="usa-accordion__button" aria-expanded="false" aria-controls="appsec">
-      Application Security and Security Operations
-    </button>
-  </h4>
-    <div id="appsec" class="usa-accordion__content">
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.kp7l6vfz1yhg">Tell me about a secure development practice (or tool) you helped introduce to an engineering team.</a></p>
-
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.tzmgeavlz1n8">Tell me about a security-related topic that you helped someone from a non-security background learn.</a></p>
-    </div>
-  <h4 class="usa-accordion__heading">
-    <button class="usa-accordion__button" aria-expanded="false" aria-controls="pentest">
-      Penetration Testing
-    </button>
-  </h4>
-    <div id="pentest" class="usa-accordion__content">
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.7x7o7oomtxsw">Tell me about a particularly interesting vulnerability that you discovered.</a></p>
-
-        <p><a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.9egaga9vhkzm">Tell me about a time when you reported an issue you discovered to a development team.</a></p>
-    </div>
-</div>
+* ["Pick a foundational web development concept or technology and explain it at two levels: first as you would to a colleague who’s not a software developer, like a designer or product manager; next, as you would to a peer."](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.rnjhk6kct2y)
+
+* ["Tell me about your approach to software testing. Why is it important and when have you done it? Where does it belong in the process?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.8vmh70tr7j6b)
+
+* ["What does refactoring mean to you? Why is it important and when have you done it? Where does it belong in the process?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.iiceusfwxdz6)
```

Comment:
FWIW, I never loved this question

Reply:
I like this question as a way to talk about technical debt, but if that's the intent, maybe we can just ask about technical debt directly? E.g.:

> Can you tell me what technical debt means to you, and how you manage it?

Reply:
I like that better, but what's the goal of the question? What do we hope to learn? What are the things to watch for and not watch for? (Of course, that may all be in the google doc I haven't looked at)

Reply:
This is sort of a process question. Sometimes the candidate will answer along the lines of "oh, we only refactor when the PM tells us to," or "once every year we have a refactoring sprint," or "I start refactoring as soon as I clone a new codebase" all of which are indicative of ways of working that just aren't compatible with how we work. I like the idea of maybe adding a follow-up about technical debt!